managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -1823,8 +1825,10 @@ void internalFlushPendingMarkDeletes() {
 
     void createNewMetadataLedger(final VoidCallback callback) {
         ledger.mbean.startCursorLedgerCreateOp();
+        final long now = System.currentTimeMillis();
```

Comment:
Actually, we are recording zk-stat mainly at two places
1. [Bookkeeper-ledger create/open](https://github.com/rdhabalia/pulsar/blob/b39f318c998face61b7c5c28731bd37951aef283/managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java#L1831) in ManagedLedger/ManagedCursor: we use bkClient and we can't modify bkClient so, we do in ManagedLedger/Cursor level.

2. Update managed-ledger data in [MetaStore](https://github.com/rdhabalia/pulsar/blob/b39f318c998face61b7c5c28731bd37951aef283/managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/MetaStoreImplZookeeper.java#L145): which directly uses zk-client so, we do at MetaStore level.

Also `ZkUtils` is part of bk client.